bin/gwdetchar-channel-diff

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
```

Comment:
@andy, please add the coding statement and the standard GPLv3 license header after the hashbang. You can copy directly from any of the other `/bin/gwdetchar-*` executables